frontend/src/pages/TermPlanner/TermBox/TermBox.tsx

```diff
@@ -19,9 +19,10 @@ type Props = {
   coursesList: string[];
   termsOffered: string[];
   dragging: boolean;
+  multiCourseDrag: string;
```

Comment:
we can just call this smth like `showMultiCourseBadge` and make this a boolean instead rather than the need to compare it as a course code string?